lib/attribrutex.ex

```diff
@@ -96,5 +97,74 @@ defmodule Attribrutex do
   defp select_custom_fields(query, :keys), do: from c in query, select: c.key
   defp select_custom_fields(query, :fields), do: from c in query, select: %{key: c.key, type: c.field_type}
 
+  @doc """
+  Use it to manage custom_fields on building the changesets.
+  You can use the opts like on `list_custom_fields_for/2` to set the
+  `context_id` and `context_type`
+
+  ## Example:
+
+    ```
+    defmodule AttribrutexUser do
+      use Ecto.Schema
+      import Ecto.Changeset
+
+      schema "users" do
+        field :email, :string
+        field :custom_fields, :map, default: %{}
+
+        timestamps()
+      end
+
+      def changeset(struct, params \\ %{}) do
+        struct
+        |> cast(params, [:email])
+        |> validate_required([:email])
+        |> Attribrutex.prepare_custom_fields(params, opts)
```

Comment:
The `opts` variable is never defined inside the function. We should either define it or remove it from the call.